.github/workflows/unittest.yml

```
@@ -0,0 +1,34 @@
+name: Python package
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.6','3.7', '3.8']
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install tox tox-gh-actions
+          pip install mock
```

I think we don't need `tox-gh-actions` yet and `mock` will be installed by tox later:
```suggestion
          pip install tox
```